libraries/botbuilder-dialogs-adaptive/src/recognizers/crossTrainedRecognizerSet.ts

```diff
@@ -19,11 +21,19 @@ const deferPrefix = 'DeferToRecognizer_';
  * Recognizer for selecting between cross trained recognizers.
  */
 export class CrossTrainedRecognizerSet extends Recognizer {
+    public static $kind = 'Microsoft.CrossTrainedRecognizerSet';
+
     /**
      * Gets or sets the input recognizers.
      */
     public recognizers: Recognizer[] = [];
 
+    public get converters(): Converters<CrossTrainedRecognizerSet> {
+        return {
+            recognizers: (resourceExplorer: ResourceExplorer) => new RecognizerListConverter(resourceExplorer),
```

Comment:
I've seen this same style of converter construction a few times. I wonder if we could express the value in the Converters type as:

```typescript
Converter<From, To> | new (...args: unknown[]): Converter<From, To>
```

and then these could all just be written as:

```typescript
return {
  recognizers: RecognizerListConverter
}
```

This would likely require some other code changes though so it's possible it would break backwards compat.